client/my-sites/plans/current-plan/index.jsx

```diff
@@ -133,25 +134,29 @@ class CurrentPlan extends Component {
 					/>
 				) }
 
-				<CurrentPlanHeader
-					selectedSite={ selectedSite }
-					isPlaceholder={ isLoading }
-					title={ title }
-					tagLine={ tagLine }
-					currentPlanSlug={ currentPlanSlug }
-					currentPlan={ currentPlan }
-					isExpiring={ isExpiring }
-					isAutomatedTransfer={ isAutomatedTransfer }
-					includePlansLink={ currentPlan && isFreeJetpackPlan( currentPlan ) }
-				/>
-				{ isEnabled( 'jetpack/checklist' ) &&
-					isJetpack &&
-					! isAutomatedTransfer && (
-						<Fragment>
-							<QueryJetpackPlugins siteIds={ [ selectedSiteId ] } />
-							<JetpackChecklist />
-						</Fragment>
-					) }
+				{ this.props.showThankYou ? (
```

Comment:
Seems like here we don't care at all whether this is a Jetpack and non-Atomic site. Do we want this as an extra check, as this is a Jetpack-site only feature?

Reply:
Interesting case… as I worked on this I realized that it need not be specific to any site type and we can render based on the URL param.

However, there is language _in_ the header related to plan setup and the checklist that is only rendered for Jetpack (non-Atomic) sites. Maybe we should include more checks here now.